### DISCUSSION TITLE
Classification of ADHD

### DISCUSSION THREAD

**Discussion by NoahTheDuke** [retextjs/retext#94] [General]
Currently, retext-equality has ADHD listed as "inconsiderate", with various "considerate" alternatives listed ([full list here](https://github.com/retextjs/retext-equality/blob/2a925cd24cc1d6260c3e2b8db9b233c65a40c13c/data/en/ablist.yml#L661-L677)). None of these are the same as [ADHD](https://en.wikipedia.org/wiki/Attention_deficit_hyperactivity_disorder), which is a real disorder that affect tens of millions of people, myself included.

I object strongly to any suggestion that a medical disorder can be replaced with cutesy words.  It's naive and insulting to suggest that ADHD can be replaced with "impulsive" or "impetuous" or even "hyperactive". I know that people can use "adhd" to mean those things but that does not outweigh its very real meaning.

**Comment by wooorm** [comment]
Hey! They are indeed not the same as ADHD.

> I know that people can use "adhd" to mean those things

This project assumes that those people use this tool.
This tool is opt-in: you can use it to proofread your thesis or blog so. Or not. And this tool assumes you know better than this tool.

If you know how to use words well, you can configure this tool to [ignore words](https://github.com/retextjs/retext-equality/tree/2a925cd24cc1d6260c3e2b8db9b233c65a40c13c#optionsignore)

The patterns are open source. Sometimes old. In this case they were added here: https://github.com/retextjs/retext-equality/pull/31.
I would appreciate it if you can contribute your suggestions.
